backend/infrahub/core/diff/calculator.py

```diff
@@ -62,8 +63,8 @@ async def calculate_diff(
             await base_diff_query.execute(db=self.db)
             for query_result in base_diff_query.get_results():
                 diff_parser.read_result(query_result=query_result)
-
-        diff_parser.parse()
+        include_unchanged = not is_first_diff
```

Comment:
Not a big deal but was thinking if `is_first_diff` could be worded differently so that we avoid this double negative situation here. Seems like it's mostly used in this location.